### PR TITLE
[CBRD-25054] Numeric type column inserted by using rownum does not show error even it's outranged.

### PR DESCRIPTION
**Pull request by beyondykk9**
http://jira.cubrid.org/browse/CBRD-25054

numeric_db_value_coerce_to_num doesn't handle overflow case for BIGINT type.

### CODE REVIEW

File: src/object/object_domain.h [hunk 1]
```diff
@@ -517,6 +519,7 @@ extern "C"
   extern int tp_enumeration_to_varchar (const DB_VALUE * src, DB_VALUE * result);
   extern int tp_domain_status_er_set (TP_DOMAIN_STATUS status, const char *file_name, const int line_no,
 				      const DB_VALUE * src, const TP_DOMAIN * domain);
+  extern int tp_get_precision (DB_VALUE * value);
```
Comment (ctshim) [inline src/object/object_domain.h]: There seems to be no need to declare it as extern.

File: src/object/object_domain.c [hunk 2]
```diff
@@ -11475,6 +11475,77 @@ tp_domain_references_objects (const TP_DOMAIN * dom)
...
+
+  static INT64 max_value[19];	/* max size of a big integer is 19 */
+  static bool init_bigint_value = false;
+
+  if (!init_bigint_value)
+    {
+      int i;
+
+      max_value[0] = 1;
+      for (i = 1; i < 18; i++)
```
Comment (ctshim) [inline src/object/object_domain.c]: Shouldn't it be "i < 19"?

Comment (beyondykk9) [reply src/object/object_domain.c]: mistake.

File: src/object/object_domain.h (hunk 1 above)
Comment (beyondykk9) [reply src/object/object_domain.h]: Ok, fix it.

**Review by hyunikn — APPROVED** [review]

**Review by ctshim — APPROVED** [review]

**Review by hornetmj — APPROVED** [review]

**Review by hgryoo — APPROVED** [review]

**Pull request merged**